### DISCUSSION TITLE
docker: latest beta

### DISCUSSION THREAD

**Discussion by riesi77** [music-assistant/support#2023] [Q&A] [answered]
Whenever there is a new beta, I need to change the following line to match the current number:
`docker pull ghcr.io/music-assistant/server:2.0.0b<nr>`
Is there a possibility to take kind of a "latest beta" so I don't have to change my docker-compose.yml each time?

**Comment by OzGav** [comment]
Have you tried `server:latest`

**Reply by marcelveldt** [reply, marked as answer]
server:beta should work

**Reply by riesi77** [reply]
Yes, it does! Thank you!

**Comment by riesi77** [comment]
Some time ago, but that would lead to  the 1.x as this is still the last release.

**Reply by OzGav** [reply]
Ok not sure then. I am only a very casual docker user.
